gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Oct 01 05:09:23 IST 2020
+#Thu Dec 17 11:27:39 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
I don't think these changes are required. Any specific reason of pushing them?

Reply:
not any reason ! I must had cared about all these and unchecked unwanted files while pusing . 
Done : deleted .idea files , added EOF in ContactPost